README.md

```diff
@@ -162,24 +172,21 @@ SELECT c.customer_id, c.source, c.region, c.member_since,
         FROM orders c LEFT OUTER JOIN customers p ON c.customer_id = p.customer_id
 ```
 
-In this example, we are setting the value `customers` to tie to the `customers_table.sql` file, 
-essentially setting up a task dependency by creating a data dependency.
+Here is a list of currently supported frontmatter argument:
```

Comment:
```suggestion
Here is a list of supported frontmatter arguments:
```
Might make sense to have this in a "Reference" section outside of the guide component of this doc.